Assets/Mirror/Editor/Weaver/Processors/TargetRpcProcessor.cs

```diff
@@ -130,6 +130,8 @@ public static MethodDefinition ProcessTargetRpcCall(WeaverTypes weaverTypes, Wri
             }
             // pass full function name to avoid ClassA.Func <-> ClassB.Func collisions
             worker.Emit(OpCodes.Ldstr, md.FullName);
+            // pass the function hash so we don't have to compute it at runtime
```

Comment:
```suggestion
            // pass the function hash so we don't have to compute it at runtime
            // otherwise each GetStableHash call requires O(N) complexity.
            // noticeable for long function names: 
            // https://github.com/MirrorNetworking/Mirror/issues/3375
```